tests/unit/test_runners.py

```
@@ -1,7 +1,12 @@
+import os
```

Flake8 (7.3.0)

[error] 'os' imported but unused

(F401)